clusterloader2/pkg/prometheus/manifests/prometheus-prometheus.yaml

```diff
@@ -9,19 +9,29 @@ metadata:
   name: k8s
   namespace: monitoring
 spec:
+  logLevel: debug
   baseImage: gcr.io/k8s-testimages/quay.io/prometheus/prometheus
   nodeSelector:
     kubernetes.io/os: linux
   replicas: 1
   resources:
     requests:
+      cpu: 200m
```

Comment:
Let's set it to the same value as limit.

Reply:
I'd say:

1. Let's set memory request and limit the same
2. Let's set cpu request depending on number of nodes but don't set the limit. Why? Quoting @thockin: 
    > Unlike memory, CPU is “compressible”. when you put a CPU limit, you artificially restrict access to otherwise idle CPU cycles, forcing things to go slow when they don't need to and causing weird issues like quota exhaustion when using more threads than cores”

Reply:
I've set memory requests/limits to same value and set only cpu requests.